style_editor.html

```diff
@@ -274,6 +274,36 @@
   height: 100vh;
   overflow-y: scroll;
 }
+.structured-view-label {
+  display: block;
+  font-size: 110%;
```

Comment:
Is this the "bold" ?
Or is the bold coming from somewhere else that I can't find?

Reply:
That would be just below 
```
/* Fat font in Structured View */
.pp span {
  padding-left: 6px;
  font-weight: 600;
}
```

Reply:
So does this make everything bold?
Why? Not a fan of over-using bold.
What does it look like?